src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings

```diff
@@ -293,3 +293,340 @@ TestLocalizationConstant = test
+
+SchemaHierarchy_SystemConfigurationFunctions = Configuration Functions
+
+SchemaHierarchy_SystemCursorFunctions = Cursor Functions
+
+SchemaHierarchy_SystemDataTypes = System Data Types
+
+SchemaHierarchy_SystemDateAndTime = Date and Time
+
+SchemaHierarchy_SystemDateAndTimeFunctions = Date and Time Functions
+
+SchemaHierarchy_SystemExactNumerics = Exact Numerics
+
+SchemaHierarchy_SystemFunctions = System Functions
+
+SchemaHierarchy_SystemHierarchyIdFunctions = Hierarchy Id Functions
+
+SchemaHierarchy_SystemMathematicalFunctions = Mathematical Functions
+
+SchemaHierarchy_SystemMetadataFunctions = Metadata Functions
+
+SchemaHierarchy_SystemOtherDataTypes = Other Data Types
+
+SchemaHierarchy_SystemOtherFunctions = Other Functions
+
+SchemaHierarchy_SystemRowsetFunctions = Rowset Functions
+
+SchemaHierarchy_SystemSecurityFunctions = Security Functions
+
+SchemaHierarchy_SystemSpatialDataTypes = Spatial Data Types
+
+SchemaHierarchy_SystemStringFunctions = String Functions
+
+SchemaHierarchy_SystemSystemStatisticalFunctions = System Statistical Functions
+
+SchemaHierarchy_SystemTextAndImageFunctions = Text and Image Functions
+
+SchemaHierarchy_SystemUnicodeCharacterStrings = Unicode Character Strings
+
+SchemaHierarchy_AggregateFunctions = Aggregate Functions
+
+SchemaHierarchy_ScalarValuedFunctions = Scalar-valued Functions
+
+SchemaHierarchy_TableValuedFunctions = Table-valued Functions
+
+SchemaHierarchy_SystemExtendedStoredProcedures = System Extended Stored Procedures
+
+SchemaHierarchy_BuiltInType = Built-in Types
+
+SchemaHierarchy_BuiltInServerRole = Built-in Server Roles
+
+SchemaHierarchy_UserWithPassword = User with Password
+
+SchemaHierarchy_SearchPropertyList = Search Property List
+
+SchemaHierarchy_SecurityPolicies = Security Policies
+
+SchemaHierarchy_SecurityPredicates = Security Predicates
+
+SchemaHierarchy_ServerRole = Server Role
+
+SchemaHierarchy_SearchPropertyLists = Search Property Lists
+
+SchemaHierarchy_ColumnStoreIndexes = Column Store Indexes
+
+SchemaHierarchy_TableTypeIndexes = Table Type Indexes
+
+SchemaHierarchy_ServerInstance = ServerInstance
+
+SchemaHierarchy_SelectiveXmlIndexes = Selective XML Indexes
+
+SchemaHierarchy_XmlNamespaces = XML Namespaces
+
+SchemaHierarchy_XmlTypedPromotedPaths = XML Typed Promoted Paths
+
+SchemaHierarchy_SqlTypedPromotedPaths = T-SQL Typed Promoted Paths
+
+SchemaHierarchy_DatabaseScopedCredentials = Database Scoped Credentials
+
+SchemaHierarchy_ExternalDataSources = External Data Sources
+
+SchemaHierarchy_ExternalFileFormats = External File Formats
+
+SchemaHierarchy_ExternalResources = External Resources
+
+SchemaHierarchy_ExternalTables = External Tables
+
+SchemaHierarchy_AlwaysEncryptedKeys = Always Encrypted Keys
+
+SchemaHierarchy_ColumnMasterKeys = Column Master Keys
+
+SchemaHierarchy_ColumnEncryptionKeys = Column Encryption Keys
+
+############################################################################
+# Extensibility
```

Comment:
>Extensibility [](start = 2, length = 13)

These are in the main Host project as far as I know. We should remove them from here and use the correct reference instead.